### DISCUSSION TITLE
Stand up Monday May 29th

### DISCUSSION THREAD

**Discussion by CarlosCRG19** [Bugs-io/TC3001B#85] [Main Stand Ups]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
- 

**What are your priorities for the next 24 hours?**:
- 

**Blockers**:
- 

**Shoutouts**:
- @username for x
```

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
- Not a lot of work tbh. On Friday, @BrandonMagana helped me to fix a problem with the retrieval of a graph

**What are your priorities for the next 24 hours?**:
- Work on architecture document.

**Blockers**:
- Depression post Chivas loss

**Shoutouts**:
- @username for x

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
- Just worked on arch doc

**What are your priorities for the next 24 hours?**:
- talk with Jacob about his graphs
- more document sh*t???

**Blockers**:
- Eye infection

**Shoutouts**:
- @chivas for turning down the monitor after the first half

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
- Helped @CarlosCRG19 debugging a communication problem between the server and analysis service. 

**What are your priorities for the next 24 hours?**:
-  Work on architecture document

**Blockers**:
-  I'm dead after chivas loss the 13th championship

**Shoutouts**:
- @CristianCazares por no ir al gym

**Comment by Jacobgtz024** [comment]
**What did you achieve in the last 24 hours?**:
- Good start on the radar graph
- Work with regular expressions for word match

**What are your priorities for the next 24 hours?**:
- Finish the radar graph data

**Blockers**:
- Perdió el futbol

**Shoutouts**:
- @Mozo

**Reply by BrandonMagana** [reply]
Pinche Mozo verga

**Reply by BrandonMagana** [reply]
Puto Dt
